domain/consensus/processes/blockvalidator/proof_of_work_test.go

```diff
@@ -15,46 +16,47 @@ import (
 	"github.com/pkg/errors"
 )
 
-// Set the flag "skip pow" to be false (second argument in the function)
+// Set the flag "skip pow" to be false (second argument in the function) for not skipping the check of POW and validate its correctness.
 func TestPOW(t *testing.T) {
```

Comment:
The convention in go is to comment functions with a comment that start with the function name "TestPOW tests that..."